go/private/rules/binary.bzl

```diff
@@ -159,6 +159,7 @@ go_tool_binary = go_rule(
         "linkstamp": attr.string(),
         "x_defs": attr.string_dict(),
         "linkmode": attr.string(values=LINKMODES, default=LINKMODE_NORMAL),
+        "_race": attr.string(default="off"),
```

Comment:
I think I make this _tool or _host, as that is really what it's a proxy for detecting, race mode is just the consequence.

Reply:
Renamed to `_hostonly`.